test/notificationSpec.js

```diff
@@ -370,10 +409,50 @@ describe('Notification handling', function () {
       }))
     })
   })
+
+  describe('notifications of incoming messages', function () {
+    it('emits "incoming_message"', function * () {
+      this.wsRedLedger.send(JSON.stringify({
+        type: 'message',
+        resource: this.fiveBellsMessage
+      }))
+
+      yield new Promise((resolve) => this.wsRedLedger.on('message', resolve))
+      sinon.assert.calledOnce(this.stubIncomingMessage)
```

Comment:
That's what the next line does: `sinon.assert.calledWith(this.stubIncomingMessage, this.message)`.